src/main/algorithms/sort/heapSort.ts

```diff
@@ -91,6 +91,19 @@ export class HeapSort implements SortAlgorithmImplementation {
     }
 
     description(): string[] {
-        return ['Heapsort description']
+        return ['Heapsort ist eine Sortierverfahren, welches auf dem binären Heap als zentrale Datenstruktur aufbaut. \
+        Im ersten Schritt verändert Heapsort die Struktur der Liste in einer solchen Form, dass ein binärer Max-Heap \
+        entsteht. In einem binären Max-Heap steht das größte Element an erster Stelle. Demnach kann das erste Element des \
```

Comment:
"In einem binären Max-Heap steht das größte Element an erster Stelle."
Eventuell nach unten zur Erklärung vom Heap schieben? Oder die Heap-Erklärung hier einfügen?